functions-and-operators/string-functions.md

```diff
@@ -1172,12 +1172,67 @@ Return a set of comma-separated strings that have the corresponding bit in bits
 
 ### [`MID()`](https://dev.mysql.com/doc/refman/8.0/en/string-functions.html#function_mid)
 
-Return a substring starting from the specified position.
+The `MID(str, start, end)` function returns a substring starting from the specified `start` position and ends at the optinal `end` position.
```

Comment:
I think `MID(str, pos, len)` is better, as the third parameter specifies the substr len, not the end position.

```sql
mysql> select mid("12345", "3", "1");
+------------------------+
| mid("12345", "3", "1") |
+------------------------+
| 3                      |
+------------------------+
```